### PR TITLE
fix compiler warnings and a string typo

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves https://musescore.org/en/node/304830 and https://musescore.org/en/node/304826

### CODE REVIEW

File: mscore/inspector/inspector_fret.ui [hunk 1]
```diff
@@ -105,16 +105,6 @@
...
-      <item row="6" column="0">
-       <widget class="QLabel" name="label_6">
-        <property name="text">
-         <string>Fret number:</string>
-        </property>
-        <property name="buddy">
-         <cstring>fretNumber</cstring>
-        </property>
-       </widget>
-      </item>
```
Comment (Spire42) [inline mscore/inspector/inspector_fret.ui:117]: Did you mean to delete this?

File: mscore/inspector/inspector_fret.ui [hunk 2]
```diff
@@ -218,22 +208,6 @@
...
-         </sizepolicy>
-        </property>
-        <property name="accessibleName">
-         <string>Fret number</string>
-        </property>
-        <property name="minimum">
-         <number>1</number>
-        </property>
-       </widget>
-      </item>
```
Comment (Spire42) [inline mscore/inspector/inspector_fret.ui:236]: Did you mean to delete this?

File: mscore/pianoroll/pianoview.cpp [hunk 3]
```diff
@@ -1865,12 +1865,12 @@ void PianoView::cutNotes()
 
 void PianoView::copyNotes()
       {
-      QString data = serializeSelectedNotes();
-      if (data.isEmpty())
+      QString dat = serializeSelectedNotes();
```
Comment (Spire42) [inline mscore/pianoroll/pianoview.cpp:1868]: How about renaming `dat` to something more meaningful, such as `copiedNotes`?

File: mscore/pianoroll/pianoview.cpp [hunk 4]
```diff
@@ -1902,10 +1902,10 @@ void PianoView::pasteNotesAtCursor()
 
       if (ms->hasFormat(PIANO_NOTE_MIME_TYPE)) {
             //Decode our XML format and recreate the notes
-            QByteArray data = ms->data(PIANO_NOTE_MIME_TYPE);
+            QByteArray dat = ms->data(PIANO_NOTE_MIME_TYPE);
```
Comment (Spire42) [inline mscore/pianoroll/pianoview.cpp:1905]: Here's another `dat` that could be renamed to `copiedNotes`.

File: mscore/pianoroll/pianoview.cpp [hunk 5]
```diff
@@ -1953,10 +1953,10 @@ void PianoView::finishNoteGroupDrag() {
 //   pasteNotes
 //---------------------------------------------------------
 
-void PianoView::pasteNotes(const QString& data, Fraction pasteStartTick, int pitchOffset, bool xIsOffset)
+void PianoView::pasteNotes(const QString& dat, Fraction pasteStartTick, int pitchOffset, bool xIsOffset)
```
Comment (Spire42) [inline mscore/pianoroll/pianoview.cpp:1956]: One more `dat` that could be renamed to `copiedNotes`.

BTW, I suggest that you also rename the parameter in the declaration of this function in the header file.

File: mscore/inspector/inspector_fret.ui (hunk 1 above)
Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_fret.ui:117]: Yes

File: mscore/inspector/inspector_fret.ui (hunk 2 above)
Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_fret.ui:236]: yes

**Comment by Jojo-Schmitz** [comment]
Travic CI failure pending the merge of PR #6023

File: mscore/pianoroll/pianoview.cpp (hunk 5 above)
Comment (Jojo-Schmitz) [reply mscore/pianoroll/pianoview.cpp:1956]: done

File: mscore/pianoroll/pianoview.cpp (hunk 4 above)
Comment (Jojo-Schmitz) [reply mscore/pianoroll/pianoview.cpp]: done

File: mscore/pianoroll/pianoview.cpp (hunk 3 above)
Comment (Jojo-Schmitz) [reply mscore/pianoroll/pianoview.cpp]: done

**Pull request merged**